### DISCUSSION TITLE
please help me to add your favorite color to list

### DISCUSSION THREAD

**Discussion by zhouyangyang369** [NAIST-SE/AttractingContributors#30] [General]
https://github.com/zhouyangyang369/AttractingContributors-color.git

**Comment by maxcell1024** [comment]
Added your color list.
Please let me know if there are any mistakes.

**Reply by duonglvtnaist** [reply]
Sorry but I think you should fork the repository from this repository. Because I found that your repository was not forked from: https://github.com/NAIST-SE/AttractingContributors

**Comment by tattiyaS** [comment]
Added a color to your list!

**Comment by vittunyuta-aom** [comment]
added!! 
https://github.com/aommoaGitHub/SuperFavoriteSongs

**Comment by noppadol-assava** [comment]
I commit pull request on your repo.
:D
